src/oal_agent/cli.py

```
@@ -65,5 +67,38 @@ def analyze(contract_file: str):
+@cli.command(help="Get the status of an analysis job.")
+@click.argument("job_id", type=str)
+def status(job_id: str):
+    """Get the status of an analysis job.
+
+    This command queries the API for the status of a specific job ID.
+
+    """
+    api_url = f"http://{settings.api_host}:{settings.api_port}/analysis/{job_id}"
+    click.echo(f"Fetching status for job ID: {job_id} from {api_url}...")
+    try:
+        response = requests.get(api_url)
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)